src/Stream.php

```
@@ -393,4 +396,36 @@ public function isPipe(): bool
 
         return $this->isPipe;
     }
+
+    /**
+     * Reads stream into a string.
+     *
+     * @throws RuntimeException
+     *
+     * return string The contents
+     */
+    protected function getStreamContents(): string
+    {
+        $contents = false;
+        $exception = null;
+
+        set_error_handler(static function ($type, $message) use (&$exception) {
+            throw $exception = new RuntimeException('Unable to read stream contents: ' . $message);
+        });
+
+        try {
+            $contents = stream_get_contents($this->stream);
```

I think that the GitHub Action error may be silenced, as `$this->resource` is always a `resource`, after class have been initialized: https://github.com/slimphp/Slim-Psr7/pull/294/files#diff-34becb2567ac761900b734bbb8adc6430566ffcb8163393b47d55ff5f2f06a06L75

Strange that the GH Action didn't pick it up before